content/c-sharp/concepts/strings/terms/replace/replace.md

```diff
@@ -0,0 +1,74 @@
+---
+Title: '.Replace()' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Returns a new string where every instance of a certain character or string in the original string is swapped with a different specified character or string.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Strings'
+  - 'Methods'
+  - 'Characters'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`.Replace()`** method when applied to the string gives back a new string where every instance of a certain character or string in the original string is swapped with a different specified character or string.
+
+## Syntax
+
+```
+// Replaces old character with new character.
+myString.Replace(char oldChar, char newChar);
+
+// Replaces old string with new string.
+myString.Replace(string oldString, string newString);
+```
+
+The `.Replace()` method takes the original string returns a new string with the following changes in it:
+
+* All occurances of indicated character are replaced with another one.
+* All occurances of indicated string are replaced with another one.
+
+> **Note:** If `oldChar` or `oldString` is not found in the original string, than the `.Replace()` method returns the string witout changes.
+
+> **Note:** If `newChar` or `newString` is `null`, than all occurances of `oldChar` or `oldString` are removed.
+
+> **Note:** The `.Replace()` method does not modify the original `string`. It always produces a new string.
+
+## Example 1
+
+The following example takes the `string` named `oldString`, replaces all the hyphen characters `-` with comma characters `,`, and returns the new `string` named `newString`.
+
+```cs
+string oldString = "A-B-C-D-E-F-G-H-I";
```

Comment:
both code examples should include a class and main call, it should be runnable as is, like a codebyte